### PR TITLE
17 move ui from task list to command

### LINKED ISSUE #17: Move UI stuff from TaskList to Command

### LINKED ISSUE #24: Parser and Command: Fix checkstyle issues

### PR DESCRIPTION
**Pull request by amanda-chua**

### CODE REVIEW

File: src/main/java/athena/ListCommand.java
Comment (daniellimws) [inline src/main/java/athena/ListCommand.java]: Need an empty line before method definition.

```
Error: eckstyle] [ERROR] /home/runner/work/tp/tp/src/main/java/athena/ListCommand.java:17: 'METHOD_DEF' should be separated from previous statement. [EmptyLineSeparator]
```

File: src/main/java/athena/AddCommand.java
Comment (daniellimws) [inline src/main/java/athena/AddCommand.java]: Need a space before `taskDuration`.

File: src/main/java/athena/AddCommand.java
Comment (daniellimws) [inline src/main/java/athena/AddCommand.java]: Need a space before `taskDuration`.

File: src/main/java/athena/EditCommand.java
Comment (daniellimws) [inline src/main/java/athena/EditCommand.java]: Need a space before `taskDuration`.

File: src/main/java/athena/EditCommand.java
Comment (daniellimws) [inline src/main/java/athena/EditCommand.java]: Need a space before `taskDuration`.

**Review by daniellimws — CHANGES_REQUESTED** [review]
Looks good. Left some comments about code quality.

I think we wait until #29 is merged before merging this so that the build doesn't break.

**Review by wish2023 — APPROVED** [review]

File: src/main/java/athena/Parser.java
Comment (daniellimws) [inline src/main/java/athena/Parser.java]: Prof said cannot using `*` to import. Refer to https://github.com/nus-cs2113-AY2021S1/forum/issues/36

**Review by daniellimws — APPROVED** [review]
Looks nice. Once you change the `import` statement I think it is good to merge.

**Pull request merged**
